### PR TITLE
feat(nimbus): Conclusion recommendations migration

### LINKED ISSUE #10668: Migrate radio recommendation data to checkbox data
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4502)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- We want to migrate conclusion recommendation data to a new field to support multi-select

This commit

- Adds a new migration to get data from `conclusion_recommendation` to `conclusion_recommendations`

Fixes #10668

### CODE REVIEW

**Comment by yashikakhurana** [comment]
Blocked on #10671

**Review by jrbenny35 — APPROVED** [review]

File: experimenter/experimenter/experiments/api/v5/types.py [hunk 1]
```diff
@@ -505,6 +505,7 @@ class NimbusExperimentType(DjangoObjectType):
     conclusion_recommendations = graphene.NonNull(
         graphene.List(graphene.NonNull(NimbusExperimentConclusionRecommendationEnum))
     )
+
```
Comment (brennie) [inline experimenter/experimenter/experiments/api/v5/types.py:508]: Undo this change

**Review by brennie — APPROVED** [review]
